Venmo/src/main/java/com/braintreepayments/api/VenmoAccount.java

```diff
@@ -9,29 +9,40 @@
     private static final String NONCE_KEY = "nonce";
 
     private String nonce;
+    private boolean shouldValidate;
 
     VenmoAccount() {}
 
-    void setNonce(String nonce) {
-        this.nonce = nonce;
-    }
-
     @Override
-    protected void buildJSON(JSONObject base, JSONObject paymentMethodNonceJson) throws JSONException {
+    JSONObject buildJSON() throws JSONException {
+        JSONObject json = super.buildJSON();
+        JSONObject paymentMethodNonceJson = new JSONObject();
         paymentMethodNonceJson.put(NONCE_KEY, nonce);
-        base.put(VENMO_ACCOUNT_KEY, paymentMethodNonceJson);
+
+        JSONObject optionsJson = new JSONObject();
+        optionsJson.put(VALIDATE_KEY, shouldValidate);
+        paymentMethodNonceJson.put(OPTIONS_KEY, optionsJson);
+
+        json.put(VENMO_ACCOUNT_KEY, paymentMethodNonceJson);
+        return json;
     }
 
-    @Override
-    protected void buildGraphQL(JSONObject base, JSONObject input) {}
+    /**
+     * @param shouldValidate Flag to denote when the associated {@link PaymentMethodNonce}
+     *                       will be validated. When set to {@code true}, the {@link PaymentMethodNonce}
+     *                       will be validated immediately. When {@code false}, the {@link PaymentMethodNonce}
+     *                       will be validated when used by a server side library for a Braintree gateway action.
+     */
+    public void setShouldValidate(boolean shouldValidate) {
+        this.shouldValidate = shouldValidate;
+    }
```

Comment:
Same here. We don't pass `validate = true` when [tokenizing Venmo](https://github.com/braintree/braintree_ios/blob/7ccda0f255230b8d427258766cb8041ba5ead5a4/Sources/BraintreeVenmo/BTVenmoDriver.m#L178) on iOS.